www/src/components/creative-layout.js

```diff
@@ -6,7 +6,9 @@ export default ({ children }) => {
   return <div className={styles.container}>
     <Header/>
     <main>
-      {children}
+      <section>
+        {children}
```

Comment:
pretty sus because you're using max-container-width on a section element and things were broken unless you did this

Reply:
this means you have nested sections extra sus